### PR TITLE
Create 2023-03-31-remote-unix-monitoring-with-net-snmp.md

### PR DESCRIPTION
**Pull request by Ferroin**
Use-case blog post about using Net-SNMP with our SNMP collector to perform basic remote monitoring for systems that we don’t support running Netdata on.

TODO list:

- [x] Basic info on SNMP
- [x] Info on basic setup of Net-SNMP
- [x] SNMP v2c authentication setup
- [x] SNMP v3 authentication setup
- [x] Basic Netdata configuraiton
- [x] Disk space monitoring info
- [x] Info about using virtual nodes with this setup
- [x] Quick blurb about using other SNMP implementations.

### CODE REVIEW

**Comment by andrewm4894** [comment]
@Ferroin love it! just tried small little reorg to use folder approach so can have an image (that live in our repo instead of an external url that might go away) for social share and stuff like that.

**Comment by andrewm4894** [comment]
actually have been wondering wth snmp is - going to read it now

File: blog/2023-03-31-remote-unix-monitoring-with-net-snmp/index.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+    _os_name: Solaris
+    _os_version: 11
+    _system_cores: 8
+    _system_cpu_freq: 4700000000
+    _system_disk_space: 1099511627776
+    _system_ram_total: 34359738368
+    _virtualization: kvm
+```
+
+Once you have a virtual node defined, you simply need to add a `vnode:` key to the SNMP job that you want to
```
Comment (andrewm4894) [inline blog/2023-03-31-remote-unix-monitoring-with-net-snmp/index.md:399]: @Ferroin wondering if/can or should we have some sort of image to show how the vnode `foo.example.com` ends up looking here in Netdata Cloud?

**Review by andrewm4894 — APPROVED** [review]
lgtm - i'd suggest post whenever and update folder name to reflect whatever date we merge this on.

File: blog/2023-03-31-remote-unix-monitoring-with-net-snmp/index.md (hunk 1 above)
Comment (Ferroin) [reply blog/2023-03-31-remote-unix-monitoring-with-net-snmp/index.md:399]: I’m not sure if it’s worth it? Virtual nodes look no different from any other node, they just have a different set of collectors listed.

**Comment by Ferroin** [comment]
> @Ferroin I think feel free to adjust date if want and merge/publish whenever (unless is some reason for 31st that i'm not aware of).
> 
> Generally publishing to blog right away is fine and then can be a different issue if/how/when we then want to share and promote it on our socials.

I’m intentionally trying to wait and publish this close to the v1.39.0 release, so that users who want the vnodes functionality but are using the stable channel don’t have to wait too long. At the moment though, we’re not actually sure when v1.39.0 will happen, just that it will probably be some time in early April, hence the choice of date.

**Comment by andrewm4894** [comment]
@Ferroin just checking - planning to bump this another week maybe or?

**Comment by Ferroin** [comment]
> @Ferroin just checking - planning to bump this another week maybe or?

Yes, given that v1.39.0 is currently expected in about two weeks. Thanks for the reminder though.

**Pull request merged**